modules/aws/vpc/vpc.tf

```diff
@@ -139,6 +139,15 @@ resource "aws_route" "vpc_local_route" {
   depends_on             = [aws_route_table.cluster_vpc_public]
 }
 
+resource "aws_route" "vpc_transit_route" {
+  count = var.transit_vpc_cidr != "" ? length(var.subnets_elb) : 0
+
+  route_table_id         = element(aws_route_table.cluster_vpc_public.*.id, count.index)
```

Comment:
Does it mean all the traffic from `cluster_vpc_{public,private}` will go trough VPN? What is there?

Reply:
No, it means that we create this rule in all routing tables. And rule itself means that traffic, where the destination is `destination_cidr_block` at line below (in our case it is transit VPC CIDR) is routed via specified gateway